CONTENT_STYLE_GUIDE.md

```diff
@@ -0,0 +1,50 @@
+# A11Y Project Content Style Guide
+General guidelines for writing articles and other content for A11Y Project.
+
+## Table of Contents
+
+1. [Spelling](#spelling)
+1. [Formatting](#formatting)
+1. [Vocabulary](#vocabulary)
+1. [Complexity](#complexity)
+1. [Tone](#complexity)
+
+## Spelling
+
+### American, UK, or Canadian Spelling
+For better or for worse, American English is generally considered to be the standard language for web developers. Let's try to use American English spelling for content on untranslated pages for consistency and clarity.
+
+### Spell-check
+Check your spelling before submitting content. Word processors usually have spell-check built-in. There are also many spell-checking websites and browser extensions.
+
+## Formatting
+Markdown tips. Heading levels. Templates?
+
+**Resources:**
+- [Online Markdown Editor](https://jbt.github.io/markdown-editor/)
+- [Markdown Cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)
+
+## Vocabulary
+### **A11Y Project**
+The name of the website is spelled with a capital A, Y, and P.
+
+### **A11Y/accessibility**
+"a11y" is a numeronym that is short for "accessibility". The number 11 stands for the 11 letters between the first and last letters of the word. Make sure to specify that the "a11y"
+
+The numeronym is to not be used in formal writing. Rather, writing out the full word, "accessibility" should be the norm. If it is used for whatever context, the abbreviation/numeronym should be defined immediately before or after with the expanded form, e.g.
+> When we talk about accessibility (A11Y), we are discussing...
+
+> The number of developers interested in A11Y (accessibility) is rising quickly...
+
+In some cases, such as the hashtag (#a11y) or a brand name (A11yProject) are acceptable for greater post exposure and brand loyalty, respectively.
+
+## Complexity
+- **Short** - Aim for a timed reading length of approximately two minutes.
+- **Focused** - Keep it digestible and to a single topic. Articles that span multiple areas and topics are better broken up.
+- **Accessible** - Strive to use plain language, avoid jargon if possible, and explain complicated concepts.
+
+**Resources:**
+- [Hemingway App](http://www.hemingwayapp.com/)
+
+## Tone
+Tone of voice tips. Professional and clear.
```

Comment:
to merge this in, we either need to flesh this section out a bit, or remove it until we have more than just this placeholder.